docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -86,26 +94,29 @@ To manage the media controller features in your application, you must learn to s
 
 To manage the media controller features in your application, you must learn to handle requests from the client to the server:
 
-1. Create a `MediaControllerChangeRequestPlaybackInfoCallback` object (in [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) and [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) applications) to define listeners for getting the requests from a client.  
+1. Create a `MediaControllerChangeRequestPlaybackInfoCallback` object (in [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) and [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) applications) to define listeners for getting the requests from a client.
```

Comment:
Suggest removing brackets from (in [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) and [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) applications).

Reason:  Removing the brackets brings a continuation in the sentence.

Suggest rephrasing to:

Create a `MediaControllerChangeRequestPlaybackInfoCallback` object in the  [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) and [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerChangeRequestPlaybackInfoCallback) applications to define listeners for getting the requests from a client.

Reply:
Global: Please verify the mobile and wearable links.

Reply:
Query to @surbhig12 
Below brackets are kind of set phrase to give the API links to each profile in Tizen docs, in case the module supports more than one profile.
 - (in mobile and wearable applications) : If the module supports mobile and wearable
 - (in mobile, wearable, and TV applications) : If the module support mobile, wearable, and TV

If you'd like to change the sentence form, please don't forget other overall guide pages for native and web also to be updated. Please always keep in mind the consistency between the pages.

Reply:
I checked links and latest directory is symbolic link to 5.5. so in my opinion this is valid link

Reply:
@mijin85cho,

Sure, I will take care of this.